src/components/Table/GridTable.tsx

```diff
@@ -544,6 +544,7 @@ function renderVirtual<R extends Kinded>(
   }, [style]);
   return (
     <Virtuoso
+      overscan={5}
```

Comment:
**question:** Is this above and below?

Reply:
Maybe in the future if we need to update this we can expose a prop?

Reply:
I believe it's above and below, but their docs don't use very clear language